src/Makefile.am

```diff
@@ -1,5 +1,9 @@
 # bin_PROGRAMS=pveclib_test
 
+# This is needed because the '#include' directives in the test suite do
+# not explicitly mention pveclib.
+AM_CFLAGS = -I$(top_srcdir)/src/pveclib/
```

Comment:
Is this really needed after this change?  Is this because the testsuite files do not include using `<pveclib/...>`?  If so, then should the testsuite files be changed?
We want the testsuite files to serve as useful examples as well.

Reply:
>Is this really needed after this change? Is this because the testsuite files do not include using <pveclib/...>?

Yes and yes.

>If so, then should the testsuite files be changed?

We can, but I wrote this patch (originally pull request #71) having Debian in mind, and Debian does not carry these files downstream, so I didn't mind.

Anyhow, I will update the pull request with this suggestion, i.e. no changes to `AM_CFLAGS`, and a lot of new changes to `#include` directives.

Reply:
Go ahead with this "last" change then we need to move on with v1.0.1...